internal/app/server/handlers/handler.go

```diff
@@ -130,17 +126,22 @@ func (h *Handler) createFromBatch(w http.ResponseWriter, r *http.Request) {
 		return
 	}
 
+	var originalURLs []string
```

Comment:
```suggestion
	originalURLs := make([]string, 0, len(urlBatch))
```
Так сильно быстрее работает.
https://t.me/bushigo/12